src/carma-platform/parse_ros2_bags.py

```diff
@@ -179,7 +179,7 @@ def read_messages(reader, topics, type_map, field_extractors):
 
 
 def filter_data_with_start_and_end_time(
-    data, topics, global_start_time, start_time, end_time
+    data, topics, global_start_time, start_time, end_time, use_relative_time=True
```

Comment:
can you add something to explain when use_relative_time should be true/false?
and with the addition of use_relative_time does the function description need to change?